modules/appeals_api/config/routes.rb

```diff
@@ -51,18 +51,11 @@
       end
 
       namespace :notice_of_disagreements do
-        resources :evidence_submissions,
-                  only: %i[create show],
-                  controller: '/appeals_api/v1/decision_reviews/notice_of_disagreements/evidence_submissions'
+        get 'contestable_issues', to: 'contestable_issues#index'
```

Comment:
We don't want contestable issues inside the NOD v2 namespace. They should use the standalone contestable issues endpoint.

Reply:
Oh, right! Omg.